projects.json

```diff
@@ -2572,6 +2572,44 @@
       }
     ]
   },
+  {
+    "repository": "Git",
+    "url": "https://github.com/louisdh/cub.git",
+    "path": "cub",
+    "branch": "master",
+    "compatibility": [
+      {
+        "version": "4.0.3",
+        "commit": "3574d3fda70091f8dea6494cafa4bca0c340b354"
+      }
+    ],
+    "platforms": [
+      "Darwin"
+    ],
+    "maintainer": "louisdhauwe@silverfox.be",
```

Comment:
Would you mind moving this "maintainer" line so that it is between the "branch" and "compatibility" fields? This way it will be consistent with the readme and other projects.